### PR TITLE
add loving logic file

### PR DESCRIPTION
**Pull request by mizzrawr**
<3

### CODE REVIEW

File: cindy/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+//c) NOT (dogIsBetter AND catIsBetter)
+console.log(!(dogIsBetter && catIsBetter));
+
+//Declare three more variables "atoms", "sandGrains", "starsInSky". Give these variables number values.
+let atoms = 10;
+let sandGrains = 20;
+let starsInSky = 5;
+
+//a) atoms is greater than starsInSky AND atoms is greater than sandGrains.
+console.log("a " + (atoms > starsInSky > sandGrains));
```
Comment (ciamac-da) [inline cindy/index.js]: 3a- The answer is incorrect! Please try my solution instead!
console.log( (atoms > starsInSky) && (atoms > sandGrains) );

File: cindy/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+//It's false only the first statement is true and the second one is false. so true * false = false.
+
+//b) atoms is NOT equal to sandGrains.
+console.log("b " + (atoms != sandGrains));
```
Comment (ciamac-da) [inline cindy/index.js]: better to use !== instead of !=

File: cindy/index.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+//It's true because It's not equal =  10 is not the same as 20
+
+//c) starsInSky is less than sandGrains OR starsInSky is greater than atoms.
+console.log("c " + (starsInSky < sandGrains || starsInSky > atoms)); //it's true because the || is taking the very first "true" result starsInSky < sandGrains.
+
+//d) atoms is equal to starsInSky OR atoms is NOT equal to sandGrains.
+console.log("d " + (atoms == starsInSky || atoms != sandGrains)); //it's true because we have the || operator. and it's rule is : it will take every true statement. even the first statement is "false", || is grabbing the second result "true"
```
Comment (ciamac-da) [inline cindy/index.js]: Here also !==

**Review by ciamac-da — CHANGES_REQUESTED** [review]

File: cindy/index.js (hunk 1 above)
Comment (mizzrawr) [reply cindy/index.js]: done it

File: cindy/index.js (hunk 2 above)
Comment (mizzrawr) [reply cindy/index.js]: done it

File: cindy/index.js (hunk 3 above)
Comment (mizzrawr) [reply cindy/index.js]: done it, thanks!

**Review by ciamac-da — APPROVED** [review]
Gracias💯

**Pull request merged**